pkgs/top-level/php-packages.nix

```diff
@@ -493,20 +493,41 @@ let
     };
   };
 
-  pthreads = assert (pkgs.config.php.zts or false); buildPecl rec {
-    version = "3.1.5";
+  pthreads = if isPhp73 then pthreads32-dev else pthreads32;
+
+  pthreads32 = assert (pkgs.config.php.zts or false); assert !isPhp73; buildPecl rec {
```

Comment:
`assert (pkgs.config.php.zts or false);`
I don’t know how to make it work correctly

Reply:
What is the resulting behavior that you need?

Reply:
If enable ztsSupport (config.php.zts or true) need build pthreads. If (pkgs.config.php.zts or false) - disable build

Reply:
But not if it's not php73 I assume?

Reply:
yes

Reply:
to php 7.3 need pthreads 3.2.0-dev